geoportailv3/static/js/themeservice.js

```diff
@@ -57,6 +83,15 @@ app.Theme = function(ngeoLocation, appThemes, appScalesService, maxExtent) {
  */
 app.Theme.prototype.setCurrentTheme = function(themeId, map) {
   this.currentTheme_ = themeId;
+
+  var piwikSiteId = this.piwikSiteIdLookup_[this.currentTheme_];
+  if (!goog.isDefAndNotNull(piwikSiteId)) {
+    piwikSiteId = this.piwikSiteIdLookup_[app.Theme.DEFAULT_THEME_];
```

Comment:
If the theme id is not found, we use the main theme.
Maybe would it be better to have a different site id ?